features/FEATURE_BLE/targets/TARGET_CORDIO/stack/ble-host/include/att_defs.h

```diff
@@ -1,22 +1,22 @@
-/*************************************************************************************************/
-/*!
- *  \file
- *
- *  \brief  Attribute protocol constants and definitions from the Bluetooth specification.
- *
- *  Copyright (c) 2009-2018 Arm Ltd. All Rights Reserved.
+/* Copyright (c) 2018 Arm Limited
```

Comment:
I think this should be 2009-2019

Reply:
I have been told on many an occasion that I'm living in the past